Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -247,11 +259,13 @@ - (void)stopAnimating {
     RunSynchronouslyOnMainThread(^{
         if (_isAnimating) {
             _isAnimating = NO;
-        }
 
-        _progressRing.IsActive(false);
-        if (_hidesWhenStopped) {
-            [self setHidden:YES];
+            if (_hidesWhenStopped) {
+                // Make our view transparent, rather than hidden, to ensure space is reserved in the layout.
+                [self setAlpha:0.0];
```

Comment:
How about leaving the Grid as-is, and simply toggling the visibility of the internal ProgressRing to Visible/Collapsed?  That would let us occupy space and show the background even when the ProgressRing is hidden.  Isn't that closer to iOS?

---
In reply to: [113288024](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113288024) [](ancestors = 113288024,113080688)